src/libraries/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -88,7 +88,7 @@ public async Task SendReceive_PartialMessageDueToSmallReceiveBuffer_Success(Uri
 
         [OuterLoop("Uses external servers")]
         [ConditionalTheory(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/45586", TestPlatforms.Browser)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/46983", TestPlatforms.Browser)]  // JS Websocket does not support see issue
```

Comment:
Do we intend to fix this?  If not it should be a platform check and not an ActiveIssue

Reply:
We can not fix this as it is.  In the issue with the JavaScript code it does not work either.  If that does not work and the network layer does not have a response there is no way to fix it in the managed code.

Reply:
Also left it as an active issue to help document why.

Reply:
Addressed in recent commits